index.d.ts

```diff
@@ -1,147 +1,71 @@
-declare module "flexsearch" {
-  export interface Index<T> {
-    readonly id: string;
-    readonly index: string;
-    readonly length: number;
-
-    init(options?: CreateOptions): this;
-    info(): {
-      id: any;
-      items: any;
-      cache: any;
-      matcher: number;
-      worker: any;
-      threshold: any;
-      depth: any;
-      resolution: any;
-      contextual: boolean;
-    };
-    add(o: T): this;
-    add(id: number, o: string): this;
-
-    // Result without pagination -> T[]
-    search(
-      query: string,
-      options: number | SearchOptions,
-      callback: (results: T[]) => void
-    ): void;
-    search(query: string, options?: number | SearchOptions): Promise<T[]>;
-    search(
-      options: SearchOptions & { query: string },
-      callback: (results: T[]) => void
-    ): void;
-    search(options: SearchOptions & { query: string }): Promise<T[]>;
-
-    // Result with pagination -> SearchResults<T>
-    search(
-      query: string,
-      options: number | (SearchOptions & { page?: boolean | Cursor }),
-      callback: (results: SearchResults<T>) => void
-    ): void;
-    search(
-      query: string,
-      options?: number | (SearchOptions & { page?: boolean | Cursor })
-    ): Promise<SearchResults<T>>;
-    search(
-      options: SearchOptions & { query: string; page?: boolean | Cursor },
-      callback: (results: SearchResults<T>) => void
-    ): void;
-    search(
-      options: SearchOptions & { query: string; page?: boolean | Cursor }
-    ): Promise<SearchResults<T>>;
-
-    update(id: number, o: T): this;
-    remove(id: number): this;
-    clear(): this;
-    destroy(): this;
-    addMatcher(matcher: Matcher): this;
-
-    where(whereObj: { [key: string]: string } | ((o: T) => boolean)): T[];
-    encode(str: string): string;
-    export(
-      callback: (key: string, data: any) => any,
-      self?: this,
-      field?: string,
-      index_doc?: Number,
-      index?: Number
-    ): Promise<boolean>;
-    import(exported: string): this;
-  }
+interface SearchOptions {
```

Comment:
why is this file so much smaller now? Is it just because of formatting? It's hard to tell what changed